Add validation case val2a (#189)
idaholab/TMAP8: doc/content/verification_and_validation/val-2a.md

## Model Description

In this case, TMAP8 simulates a one-dimensional domain to represent the deuterium diffusion. The deuterium is generated by ion beam and leave from upstream and downstream side by recommendation. Note that this case can easily be extended to a two- or three-dimensional case.

The source term in the model is described as a normal distribution instead of a piecewise function from TMAP4. The source term of deuterium versus the coordinate is defined as:

\begin{equation} \label{eq:normal_distribution}
S = \frac{1.5}{\sigma \sqrt(2 \pi) \exp \left( -0.5 * ((x - \mu ) / \sigma)^2 \right)},
\end{equation}

where $\sigma$ is the characteristic width of the normal distribution, and $\mu$ is the depth of the normal distribution from the upstream side. The comparison between the normal distribution from TMAP8 and piecewise function from TMAP4 is shown in [val-2a_normal_distribution]. The normal distribution has the similar distribution with the piecewise function, and also provide a smooth distribution, which is close to the real situation.

> !media figures/val-2a_normal_distribution.png

Reviewer: Please add units for y axis.

\begin{equation} \label{eq:recommendation_ignore_Pressure}
J = A K_r C^2.
\end{equation}

## Case and Model Parameters

The beam flux on the upstream side of the sample during the experiment are presented in [val-2a_flux_and_pressure_TMAP4]. Other case and model parameters in TMAP8 are listed in [val-2a_set_up_values_TMAP4]. Some of the parameters are directly leveraged from [!cite](anderl1985tritium,longhurst1992verification,ambrosek2008verification), but others were adapted to better match the experimental data.

!table id=val-2a_flux_and_pressure_TMAP4 caption=Values of beam flux on the upstream side of the sample during the experiment [!citep](longhurst1992verification).
| time (s)      | Beam flux (atom/m$^2$/s)     |
| ---------     | ---------------------------- |
| 0 - 5820      | 4.9$\times 10^{19}$          |
| 5820 - 9056   | 0                            |
| 9056 - 12062  | 4.9$\times 10^{19}$          |
| 12062 - 14572 | 0                            |
| 14572 - 17678 | 4.9$\times 10^{19}$          |
| 17678 - 20000 | 0                            |

!alert warning title=Typo in [!cite](longhurst1992verification)

> The times of starting and stopping the beam from TMAP4 in [!cite](longhurst1992verification) are not accurate, TMAP8 uses the times from [!cite](anderl1985tritium) directly to get better fit on results.

Reviewer: ```suggestion
The times listed in [!cite](longhurst1992verification) for TMAP8 for the starts and ends the beam are not accurate. instead, TMAP8 uses the times directly from [!cite](anderl1985tritium) to better correspond to experimental conditions.
```

> !table id=val-2a_set_up_values_TMAP4 caption=Values of material properties.

Reviewer: A a reference column and add the reference used for each parameter value whenever you found it somewhere else.

| Parameter | Description                          | Value                                                       | Units                 |
| --------- | ------------------------------------ | ----------------------------------------------------------- | --------------------- |

> | $K_{d,l}$ | dissociation coefficient upstream       | 8.959 $\times 10^{18} (1-0.9999 \exp(-6 \times 10^{-5} t))$ | at/m$^2$/s/Pa$^{0.5}$ |
> | $K_{d,r}$ | dissociation coefficient on downstream  | 1.7918$\times 10^{15}$                                      | at/m$^2$/s/Pa$^{0.5}$ |
> | $K_{r,l}$ | recombination coefficient on upstream   | 1$\times 10^{-27} (1-0.9999 \exp(-6 \times 10^{-5} t))$     | m$^4$/at/s            |
> | $K_{r,r}$ | recombination coefficient on downstream | 2$\times 10^{-31}$                                          | m$^4$/at/s            |

Reviewer: ```suggestion
| $K_{d,l}$ | upstream dissociation coefficient       | 8.959 $\times 10^{18} (1-0.9999 \exp(-6 \times 10^{-5} t))$ | at/m$^2$/s/Pa$^{0.5}$ |
| $K_{d,r}$ | downstream dissociation coefficient   | 1.7918$\times 10^{15}$                                      | at/m$^2$/s/Pa$^{0.5}$ |
| $K_{r,l}$ | upstream recombination coefficient    | 1$\times 10^{-27} (1-0.9999 \exp(-6 \times 10^{-5} t))$     | m$^4$/at/s            |
| $K_{r,r}$ | downstream recombination coefficient  | 2$\times 10^{-31}$                                          | m$^4$/at/s            |
```

> | $P_{l}$   | pressure on downstream               | 0                                                           | Pa                    |
> | $P_{r}$   | pressure on downstream               | 0                                                           | Pa                    |

Reviewer: ```suggestion
| $P_{l}$   | upstream pressure               | 0                                                           | Pa                    |
| $P_{r}$   | downstream pressure               | 0                                                           | Pa                    |
```

| $D$       | deuterium diffusivity in PCA         | 3$\times 10^{-10}$                                          | m$^2$/2               |
| $d$       | diameter of PCA                      | 0.025                                                       | m                     |
| $l$       | thickness of PCA                     | 5$\times 10^{-4}$                                           | m                     |
| $T$       | temperature                          | 703                                                         | K                     |

> !alert note title=This validation case replicate TMAP4 rather than TMAP7 due to inconsistent experiment results with [!cite](anderl1985tritium).
> TMAP4 [!citep](longhurst1992verification) and TMAP7 [!citep](ambrosek2008verification) both replicate this validation case. However, they use different model parameters and configurations, and the experimental data presented in the experimental data in [!citep](ambrosek2008verification) for TMAP7 do not correspond to the data published in [!cite](anderl1985tritium). We therefore replicate only the data from TMAP4 in this TMAP8 validation case.

Reviewer: ```suggestion
!alert note title=This validation case replicates TMAP4 rather than TMAP7 due to inconsistent experiment results with [!cite](anderl1985tritium).
TMAP4 [!citep](longhurst1992verification) and TMAP7 [!citep](ambrosek2008verification) both replicate this validation case. However, they use different model parameters and configurations, and the experimental data presented in [!citep](ambrosek2008verification) for TMAP7 do not correspond to the data published in [!cite](anderl1985tritium). We therefore replicate only the data from TMAP4 and [!cite](anderl1985tritium) in this TMAP8 validation case.
```

> !alert note title=The recombination and dissociation coefficient on upstream are simple exponential.
> Both TMAP4 [!citep](longhurst1992verification) and TMAP7 [!citep](ambrosek2008verification) simplify the recombination and dissociation coefficients on upstream as exponential in time rather than an integrated ion influence in experiment. Thus, TMAP8 uses the same exponential on recombination and dissociation coefficients.

Reviewer: ```suggestion
!alert note title=The upstream recombination and dissociation coefficients as time-dependent exponentials.
Both TMAP4 [!citep](longhurst1992verification) and TMAP7 [!citep](ambrosek2008verification) describe the upstream recombination and dissociation coefficients as time-dependent exponentials rather than mechanistically capture the influence of ion irradiation on material performance. TMAP8 uses the same expressions.
```

\begin{equation} \label{eq:recombination_ignore_Pressure}
J = 2 A K_r C^2.
\end{equation}

## Case and Model Parameters

The beam flux on the upstream side of the sample during the experiment is presented in [val-2a_flux_and_pressure_TMAP4], and only 75 % of the flux remain in the sample. Other case and model parameters used in TMAP8 are listed in [val-2a_set_up_values_TMAP4].

!table id=val-2a_flux_and_pressure_TMAP4 caption=Values of beam flux on the upstream side of the sample during the experiment [!citep](longhurst1992verification).
| time (s)      | Beam flux $F$ (atom/m$^2$/s)   |
| ---------     | ---------------------------- |
| 0 - 5820      | 4.9$\times 10^{19}$          |
| 5820 - 9056   | 0                            |
| 9056 - 12062  | 4.9$\times 10^{19}$          |
| 12062 - 14572 | 0                            |
| 14572 - 17678 | 4.9$\times 10^{19}$          |
| 17678 - 20000 | 0                            |

!alert warning title=Typo in [!cite](longhurst1992verification)

> The times listed in [!cite](longhurst1992verification) for TMAP8 for the starts and ends the beam are not accurate. instead, TMAP8 uses the times directly from [!cite](anderl1985tritium) to better correspond to experimental conditions.

Reviewer 2: ```suggestion
The times listed in [!cite](longhurst1992verification) for TMAP8 for the start and end times of the beam are not accurate. instead, TMAP8 uses the times directly from [!cite](anderl1985tritium) to better correspond to experimental conditions.
```

\begin{equation} \label{eq:recommendation}
J = A (K_r C^2 - K_d P),
\end{equation}

where $J$ is the recombination flux out of the sample sides, $A$ is the area on the upstream or downstream side, $P$ is the pressure on the corresponding side, and $K_r$ and $K_d$ are the recombination and dissociation coefficients, respectively.

The objective of this simulation is to determine the permeation flux on the downstream side and match the experimental data published in [!cite](anderl1985tritium) and reproduced in [val-2a_comparison].

## Model Description

In this case, TMAP8 simulates a one-dimensional domain to represent the deuterium implantation, diffusion, and recombination. Note that this case can easily be extended to a two- or three-dimensional case.

The source term in the model is described as a normal distribution instead of the piecewise function from TMAP4 [!citep](longhurst1992verification). The source term of deuterium from ion beam implantation is defined as:

\begin{equation} \label{eq:normal_distribution}

> S = F \frac{1.5}{\sigma \sqrt(2 \pi)} \exp \left( - \frac{(x - \mu )^2}{2 \sigma^2} \right),

Reviewer: ```suggestion
S = F \frac{1.5}{\sigma \sqrt{2 \pi}} \exp \left( - \frac{(x - \mu )^2}{2 \sigma^2} \right),
```

> S = F \frac{1.5}{\sigma \sqrt(2 \pi)} \exp \left( - \frac{(x - \mu )^2}{2 \sigma^2} \right),

Reviewer: Why the 1.5 factor in the nominator? Maybe explain in the text below.

Author: The factor of 1.5 is from TMAP4, I will explain it on documentation.

\end{equation}

> where $F$ is the implantation flux, $\sigma = 2.4 \times 10^{-9}$ m is the characteristic width of the normal distribution, and $\mu = 14 \times 10^{-9}$ m is the depth of the normal distribution from the upstream side. The comparison between the normal distribution from TMAP8 and piecewise function from TMAP4 is shown in [val-2a_normal_distribution]. The normal distribution has a similar distribution to the piecewise function, but the distribution profile is closer to the expected implantation profile.

Reviewer: ```suggestion
where $F$ is the implantation flux provided in [val-2a_flux_and_pressure_TMAP4], $\sigma = 2.4 \times 10^{-9}$ m is the characteristic width of the normal distribution, and $\mu = 14 \times 10^{-9}$ m is the depth of the normal distribution from the upstream side. The comparison between the normal distribution from TMAP8 and piecewise function from TMAP4 is shown in [val-2a_normal_distribution]. The normal distribution has a similar distribution to the piecewise function, but the distribution profile is closer to the expected implantation profile.
```

!media comparison_val-2a.py
       image_name=val-2a_comparison_normal_distribution.png
       style=width:50%;margin-bottom:2%;margin-left:auto;margin-right:auto
       id=val-2a_normal_distribution
       caption= Comparison between the normal distribution from TMAP8 and piecewise function from TMAP4 for the source term due to deuterium ion beam implantation.

> The pressures on the upstream and downstream sides are close to vacuum pressures, and have only little impact for the recombination on both sides [!citep](longhurst1992verification,ambrosek2008verification). Thus, TMAP8 ignores the impact of pressure on the boundary conditions to simplify the model. The recombination is described asa simplified version of [eq:recommendation]:

Reviewer: ```suggestion
The pressures on the upstream and downstream sides are close to vacuum pressures, and have only little impact for the recombination on both sides [!citep](longhurst1992verification,ambrosek2008verification). Thus, TMAP8 ignores the impact of pressure on the boundary conditions to simplify the model. The recombination is described a sa simplified version of [eq:recommendation]:
```

\begin{equation} \label{eq:recommendation_ignore_Pressure}
J = A K_r C^2.
\end{equation}

## Case and Model Parameters

> The beam flux on the upstream side of the sample during the experiment is presented in [val-2a_flux_and_pressure_TMAP4], and only 75 % of the flux remain in the sample. Other case and model parameters used in TMAP8 are listed in [val-2a_set_up_values_TMAP4]. Some of the parameters are directly leveraged from [!cite](anderl1985tritium,longhurst1992verification,ambrosek2008verification), but others were adapted to better match the experimental data.

Reviewer: ```suggestion
The beam flux on the upstream side of the sample during the experiment is presented in [val-2a_flux_and_pressure_TMAP4], and only 75 % of the flux remain in the sample. Other case and model parameters used in TMAP8 are listed in [val-2a_set_up_values_TMAP4].
```
If you use values from TMAP4 and TMAP7, then this sentence can disappear. Otherwise, please specify what values are updated.

!table id=val-2a_flux_and_pressure_TMAP4 caption=Values of beam flux on the upstream side of the sample during the experiment [!citep](longhurst1992verification).

> | time (s)      | Beam flux F (atom/m$^2$/s)   |

Reviewer: ```suggestion
| time (s)      | Beam flux $F$ (atom/m$^2$/s)   |
```
